replicator/src/main/scala/com/evolutiongaming/kafka/journal/replicator/commit/AsyncPeriodicTopicCommit.scala

```diff
@@ -0,0 +1,204 @@
+package com.evolutiongaming.kafka.journal.replicator.commit
+
+import cats.Applicative
+import cats.data.{NonEmptyMap, NonEmptySet}
+import cats.effect.*
+import cats.effect.implicits.*
```

Comment:
```suggestion
```

Reply:
I won't compile - `.toResource` is used in the code here

Reply:
right, `import import cats.effect.syntax.all.*` is the way to go

Reply:
fixed